crates/sui-bridge-cli/src/main.rs

```diff
@@ -431,3 +512,73 @@ async fn main() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[derive(serde::Serialize, Default)]
+struct PrintEthBridge {
```

Comment:
this is a bit of a strange name to me, but if that is ok with you it's all good. Some for the other "print"...
`EthBridgeSummary`? (not sure that is a good name but felt I had to try something else - lol)

Reply:
will update

Reply:
renamed to `Output`